modules/core/core-api/src/main/java/com/enonic/xp/node/NodeHits.java

```diff
@@ -34,6 +35,14 @@ public NodeIds getNodeIds()
         return NodeIds.from( this.stream().map( NodeHit::getNodeId ).collect( Collectors.toList() ) );
     }
 
+    public NodePaths getNodePaths()
+    {
+        return NodePaths.from( this.stream().
+            map( NodeHit::getNodePath ).
+            filter( Objects::nonNull ).
```

Comment:
I would prefer NPE here instead, because it is incorrect to not request paths, and ask for them later